app/src/main/java/de/tum/in/tumcampusapp/utils/ContactsManager.kt

```diff
@@ -0,0 +1,188 @@
+package de.tum.`in`.tumcampusapp.utils
+
+import android.content.ContentProviderOperation
+import android.content.Context
+import android.content.OperationApplicationException
+import android.graphics.Bitmap
+import android.os.RemoteException
+import android.provider.ContactsContract
+import de.tum.`in`.tumcampusapp.R
+import de.tum.`in`.tumcampusapp.component.tumui.person.model.Contact
+import de.tum.`in`.tumcampusapp.component.tumui.person.model.Employee
+import java.io.ByteArrayOutputStream
+import java.io.IOException
+import java.util.*
+
+class ContactsManager {
```

Comment:
Could we please rename this to `ContactHelper` as we have `Managers` already used in this project for controller like classes.

Reply:
Fixed